docs/dialect.md

```diff
@@ -159,7 +159,7 @@ Allowable date parts: year, quarter, month, dayofyear, day, weekday, week, hou
  The value for this property can be:
 
   - __NoNullCheck__
-      Does not check that the value of LHS and RHS is null. Logic: `(lhs [!]= rhs)`
+      Does not check that the value of LHS and RHS is null. Logic: `(lhs [!]= rhs)` . Note : A known limitation is that using this value will cause Tableau to give the wrong answer in some cases, if the column is ever null. It is advised not to use this value unless there is no other option.
```

Comment:
```suggestion
      Does not check that the value of LHS and RHS is null. Logic: `(lhs [!]= rhs)` . Note: A known limitation is that using this value will cause Tableau to give the wrong answer in some scenarios if the column contains null. This is not recommended unless required by the underlying database.
```

Someone else can please double check my wording too, thanks

Another way to make this very clear, make it the last in the list instead of first, and change the example on line 177 to `Operator`.

Reply:
Fixed